### DISCUSSION TITLE
Experiencing 502 Bad Gateway on GitHub Education platform 📢

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#44483] [GitHub Education] [answered]
Hi Everyone! 👋

We want to express our thanks for your ongoing patience and understanding of the intermittent 502 errors that students and teachers are currently receiving on the GitHub Education platform. We apologise for the inconveniences you're facing when accessing GitHub Education. The team is currently working hard to resolve these errors and will be able to resume services soon.

If you do experience these 502 errors whilst navigating Global Campus or [Education Github](https://education.github.com/) please comment here so we can help assist the internal team further.

We can provide a workaround that may resolve your issues by clearing your browser’s cache and visiting https://education.github.com/globalcampus/student. Please let us know if this helps you with navigating GitHub Education platform thereafter.

Thank you again! We will post an update with further details when available. 👍

**Comment by divais** [comment, marked as answer]
Hi everyone! 👋

We want to thank you all for your patience and using the workaround while we resolved the 502 errors for students and teachers on [education.github.com](education.github.com). We are still monitoring but are happy to announce that the issue is resolved.  

If you are still experiencing these 502 error please do reply here and provide any information that would be relevant.

We appreciate your continued support! 

Thanks,
GitHub Education.
